packages/addons/addon-depends/rpi-tools-depends/gpiod/package.mk

```diff
@@ -0,0 +1,24 @@
+# SPDX-License-Identifier: GPL-2.0
+# Copyright (C) 2016-present Team LibreELEC (https://libreelec.tv)
+
+PKG_NAME="gpiod"
+PKG_VERSION="2.2.2"
+PKG_SHA256="aa4571f64e1070d38e4c14c491f69637ee73e23c801d3a418dde4d5410e5d53f"
+PKG_ARCH="arm aarch64"
+PKG_LICENSE="LGPLv2.1"
+PKG_SITE="https://pypi.org/project/gpiod/"
+PKG_URL="https://files.pythonhosted.org/packages/5a/be/9876843be7d8d3e05582de8919077b9084beac5d9791b139653e3048e4fd/gpiod-${PKG_VERSION}.tar.gz"
```

Comment:
also available at https://files.pythonhosted.org/packages/source/g/gpiod/gpiod-2.2.2.tar.gz

Reply:
Thank you. Your advice is very welcome. I was looking for a more general URL without the hash value of a CDN. Do you know the pattern from the past or is there an overview I missed?

Reply:
> Do you know the pattern from the past or is there an overview I missed?

I googled the filename and this returned some Yocto configurations with a better URL format.